documentation/service-rules/school-admins.md

```diff
@@ -2,89 +2,103 @@
+### Number of users that can access the service
 
 We will not limit the number of users per school. This means multiple people can get access, unlike in ECF1. Schools will manage the users who can access the service for their school themselves via DfE Sign In.
 
 You can read more about [why we made this decision in our design history](https://teacher-cpd.design-history.education.gov.uk/ecf-v2/allowing-multiple-school-accounts/).
 
-## Registering an ECT
+<a id="giving_details_for_an_ect"></a>
 
-### Finding an ECT's record in the Teaching Record System
+## Giving details for an ECT
 
-When a school registers an ECT, we need to find the ECT's TRS record so that we can check they are a real person and so that we can determine their eligibility for training.
+### Finding an ECT’s record in the Teaching Record System
 
-To find an ECT's teacher record, we need to check the Teaching Record System (TRS) API. The ECT needs a TRN and a teacher record to be eligible for training.
+When a school registers an ECT, we need to find the ECT’s TRS record so that we can check they are a real person, that the school knows them and so that we can determine their eligibility for training.
 
-To do this, we ask for the ECT's TRN and date of birth.
+To find an ECT’s teacher record, we need to check the Teaching Record System (TRS) API. The ECT needs a TRN and a teacher record to be eligible for training.
 
-We should always ask for two fields of personal information before registering an ECT's record. This is because we need to make sure the person registering the ECT actually knows that ECT. If we just ask for TRN, they could be entering a random number.
+To do this, we ask for the ECT’s TRN and date of birth.
 
-If we can't find a matching TRN that exists, we tell the user the ECT's teacher record cannot be found.
+We should always ask for two fields of personal information before registering an ECT’s record. This is because we need to make sure the person registering the ECT actually knows that ECT. If we just ask for TRN, they could be entering a random number.
 
-If we can't find the ECT's record in the Teaching Record System API, but the TRN does exist, we ask for that ECT's national insurance number instead. We do this because potentially the date of birth stored on the teacher's record in the TRS is incorrect, or the one the school holds might be. It makes it more likely for the user to find the ECT's teacher record and confirm they know them.
+If we can’t find a matching TRN that exists, we tell the user the ECT’s teacher record cannot be found.
+
+If we can’t find the ECT’s record in the Teaching Record System API, but the TRN does exist, we ask for that ECT’s national insurance number instead. We do this because potentially the date of birth stored on the teacher’s record in the TRS is incorrect, or the one the school holds might be. It makes it more likely for the user to find the ECT’s teacher record and confirm they know them.
```

Comment:
```suggestion
If we can find a record in TRS matching the ECT's TRN but not the provided date of birth, we ask for the ECT's national insurance number instead.
We do this because potentially the date of birth stored on the teacher’s record in the TRS is incorrect, or the one the school holds might be. It makes it more likely for the user to find the ECT’s teacher record and confirm they know them.
```